extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1316,7 +1333,7 @@ static public function getWikiByID( $id, $master = false ) {
 	 * @return array an array of objects, keys are wikis ids.
 	 */
 	static public function getWikisByID( array $ids, $master = false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)